### PR TITLE
Boundary Inclusion Requirement

### PR DESCRIPTION
**Pull request by Henry-WattTime**
Established the boundary requirement. Based on discussion in issue #56

### CODE REVIEW

**Comment by Henry-WattTime** [comment]
Should 'software boundary' be a term in the dictionary?

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -133,9 +133,11 @@ All actions that can reduce the carbon emissions of a piece of software fall int
 
 This standard can be used to calculate the real-world emissions associated with software by measuring the total change in global emissions associated with a particular piece of software.
 
+The calculation of software carbon intensity must include include all supporting infrastructure and systems that significantly contributes to the software operation. Supporting infrastructure and systems may include compute resources, storage, networking equipment, memory, monitoring, idle machines, logging, scanning, testing, operations, and backup. The entity calculating software carbon intensity must report what is included within this boundary.
```
Comment (vaughanknight) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:136]: doubled up on a word "must include include"

**Comment by vaughanknight** [comment]
It's looking good.  I won't be able to make the weeklly call so putting it in a comment here

**Comment by atg-abhishek** [comment]
@Henry-WattTime the term you requested has been added [here](https://github.com/Green-Software-Foundation/Dictionary/issues/25) as an issue.

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
@Henry-WattTime it looks good but we need minor changes: the words must and may should be capitalized so that we indicate that they carry a very specific connotation as specified in the dictionary and the references for the SCI spec itself.

**Comment by atg-abhishek** [comment]
oh and @Henry-WattTime actually it might be a good idea to bullet the list of items included to make them more evident rather than having them as a part of a single sentence.

**Comment by dtoakley** [comment]
Agree with both capitalizing the MUST and MAY, as well as listing the items in a bullet list. Other than that, this looks good!

**Comment by Henry-WattTime** [comment]
Approved by working group 2020/09/09

**Pull request merged**